modules/project-docs/pages/sdk-release-notes.adoc

```diff
@@ -34,20 +34,146 @@ include::hello-world:page$start-using-sdk.adoc[tag=install]
 
 == Upgrading to 3.x
 
-Python SDK 3.n automatically bundles libcouchbase (downloading and building it if necessary),
- so no need to install it separately.
+Python SDK 3.n automatically bundles libcouchbase (downloading and building it if necessary).
```

Comment:
Question for @RichardSmedley, here we use `3.n`, should it be `3.x`?

Reply:
We use 3.n elsewhere, and 3.x - let's discuss this in the team, and then change (or not) universally.